test/browser/spec/ol/interaction/snap.test.js

```diff
@@ -91,11 +98,18 @@ describe('ol.interaction.Snap', function () {
         coordinate: [1, 3],
         map: map,
       };
+      snapInteraction.on('snap', function (snapEvent) {
+        expect(snapEvent.type).to.be('snap');
+        expect(snapEvent.vertex).to.eql([0, 0]);
+        expect(snapEvent.vertexPixel).to.eql([width / 2, height / 2]);
+        expect(snapEvent.feature).to.be(point);
+        done();
+      });
```

Comment:
When you change the `snap` event to return the event pixel and coordinate, this test won't be necessary any more. If you decide not to make that change, the same problem as above applies.

Reply:
see comment above https://github.com/openlayers/openlayers/pull/14676#discussion_r1264607171